src/views/document/utils/field-viewers/MarkdownSection.vue

```diff
@@ -1,31 +1,31 @@
 <template>
   <div v-if="(document.cooked[field.name] && field.isVisibleFor(document)) || $slots.after" class="markdown-section">
-    <h2 v-if="field.name != 'summary' && !hideTitle" class="title is-2" :class="{ 'no-print': !visible }">
-      <span>
-        {{ (title || $gettext(field.name)) | uppercaseFirstLetter }}
-      </span>
-      <fa-icon
-        class="is-size-6 is-pulled-right has-cursor-pointer no-print"
-        icon="angle-down"
-        :rotation="visible ? undefined : 180"
-        @click="visible = !visible"
-      />
-    </h2>
-    <div v-show="visible" :lang="lang">
-      <markdown
-        v-if="document.cooked[field.name]"
-        :class="{ 'is-italic': field.name === 'summary' }"
-        :content="document.cooked[field.name]"
-      />
-      <slot name="after" />
-    </div>
+    <accordion-item>
+      <h2 slot="title" v-if="field.name != 'summary' && !hideTitle" class="title is-2">
+        <span>
+          {{ (title || $gettext(field.name)) | uppercaseFirstLetter }}
+        </span>
+      </h2>
+      <div slot="content" :lang="lang">
+        <markdown
+          v-if="document.cooked[field.name]"
+          :class="{ 'is-italic': field.name === 'summary' }"
+          :content="document.cooked[field.name]"
+        />
+        <slot name="after" />
+      </div>
+    </accordion-item>
   </div>
 </template>
 
 <script>
+import AccordionItem from '@/components/generics/AccordionItem.vue';
 import { requireDocumentProperty, requireFieldProperty } from '@/js/properties-mixins';
 
 export default {
+  components: {
```

Comment:
Thanks to https://github.com/c2corg/c2c_ui/blob/master/src/js/vue-plugins/generic-components.js, you do not need to declare the components from `generics` folder